test/molochv3.test.js

```diff
@@ -66,4 +73,33 @@ contract('MolochV3', async accounts => {
     address = await contract.getAddress(moduleId);
     assert.equal(address, "0x0000000000000000000000000000000000000000");
   });
+
+  const numberOfShares = web3.utils.toBN('1000000000000000');
+  const sharePrice = web3.utils.toBN(web3.utils.toWei("120", 'finney'));
+  const remaining = sharePrice.sub(web3.utils.toBN('50000000000000'))
+
+  async function prepareSmartContracts() {
+    let lib = await FlagHelperLib.new();
+    let bank = await BankContract.new();
+    await MemberContract.link("FlagHelper", lib.address);
+    await ProposalContract.link("FlagHelper", lib.address);
+    let member = await MemberContract.new();
+    let proposal = await ProposalContract.new();
+    let voting = await VotingContract.new();
+    let daoFactory = await DaoFactory.new(bank.address, member.address, proposal.address, voting.address);
```

Comment:
I liked this idea of Factory, so you can create a DAO with different types of contract implementations. Sweet.

Reply:
This is the role of the factory owner to make sure it's all done correctly